src/domain/gpqueue.ts

```diff
@@ -56,7 +56,7 @@ export function create(dao: Address,
        setContributionRewardParams(dao, scheme, gpAddress, parameters.value0);
        isGPQue = true;
      } else {
-       setSchemeParamsError(dao, scheme);
+       setSchemeParamsError(crypto.keccak256(concat(dao, scheme)).toHex());
```

Comment:
the schemeId can be get on the beginning of the function and rep in local var : 
let schemeId = crypto.keccak256(concat(dao, scheme)).toHex()